scripts/clonebuild.sh

```diff
@@ -29,39 +24,25 @@ DEPLOY_GIT_BRANCH=${@:$OPTIND+2:1}
 DEEP_CLEAN=${@:$OPTIND+3:1}
 NAMED_BRANCH=${@:$OPTIND+4:1}
 
+# Remove the clone folder
+if [ $DEEP_CLEAN = true ]; then
+  make cleanall
+fi
+
 create_snapshot_dir
 cd ${CLONEDIR}
 
+# Clone or update the project
 if [ ! -d mf-geoadmin3 ]; then
   echo "Cloning branch=${DEPLOY_GIT_BRANCH}, into directory=${CLONEDIR}"
   git clone -b ${DEPLOY_GIT_BRANCH}  https://github.com/geoadmin/mf-geoadmin3.git
   cd mf-geoadmin3
 else
   cd mf-geoadmin3
-  if [ "$DEEP_CLEAN" = "true" ]; then
-    make cleanall
-  else
-    make clean
-  fi
+  make clean
   update_and_reset_git_project
 fi
 
-if [ "$NAMED_BRANCH" = "true" ]; then
-  export KEEP_VERSION="true"
-  S3_BASE_PATH=/$DEPLOY_GIT_BRANCH/
-else
-  export KEEP_VERSION="false"
-  make .build-artefacts/last-version
-  S3_BASE_PATH=$(get_s3_basepath $DEPLOY_GIT_BRANCH)
-fi
-S3_SRC_BASE_PATH=$S3_BASE_PATH"src/"
-
-echo "S3_BASE_PATH:"
-echo $S3_BASE_PATH
-echo "S3_SRC_BASE_PATH:"
-echo $S3_SRC_BASE_PATH
```

Comment:
I think you could still display these. It's useful to know what's going on.

Reply:
They are set in the makefile. I'm not sure they are accessible outside

Reply:
Ok

Reply:
We can display it when we execute a  Make target.

Reply:
I've added a showVariable target in make . It is used on each beginning of build or deploy targets